django-stubs/utils/timezone.pyi

```diff
@@ -44,8 +44,8 @@ class override(ContextDecorator):
     def __enter__(self) -> None: ...
     def __exit__(self, exc_type: object, exc_value: object, traceback: object) -> None: ...
 
-def localtime(value: _AnyTime, timezone: Optional[tzinfo] = None) -> datetime: ...
-def localdate(value: _AnyTime, timezone: Optional[tzinfo] = None) -> date: ...
+def localtime(value: Optional[_AnyTime], timezone: Optional[tzinfo] = None) -> datetime: ...
+def localdate(value: Optional[_AnyTime], timezone: Optional[tzinfo] = None) -> date: ...
```

Comment:
It's actually optional argument, per https://github.com/django/django/blob/7f19e3713598a37b0809b5434114140170d12e34/django/utils/timezone.py#L211
Could you add `= ...` (and also replace `None` in the second arg with `...` too)

Reply:
Good catch! Thanks!